arches/app/templates/views/components/search/advanced-search.htm

```diff
@@ -82,18 +82,18 @@ <h4 class="search-facet-item-heading">
                         type="text" 
                         class="form-control"
                         data-bind="
-                            attr:{placeholder: $root.translations.find + '...'},
+                            attr:{placeholder: $root.translations.find + '...', 'aria-label': $root.translations.find + '...'},
```

Comment:
This label should probably say something like `search facets` --> (`$root.translations.searchFacets`)

Reply:
Changed to "Search facets"